x-pack/plugins/aiops/public/components/log_categorization/category_table/category_table.tsx

```diff
@@ -302,49 +238,49 @@ export const CategoryTable: FC<Props> = ({
     };
   };
 
-  return (
-    <>
-      <TableHeader
-        categoriesCount={categories.length}
-        selectedCategoriesCount={selectedCategories.length}
-        labels={labels}
-        openInDiscover={(queryMode: QueryMode) => openInDiscover(queryMode)}
-      />
-      <EuiSpacer size="xs" />
-      <EuiHorizontalRule margin="none" />
+  const tableStyle = css({
+    thead: {
+      position: 'sticky',
+      insetBlockStart: 0,
+      zIndex: 1,
+      backgroundColor: euiTheme.euiColorEmptyShade,
+      boxShadow: `inset 0 0px 0, inset 0 -1px 0 ${euiTheme.euiBorderColor}`,
+    },
+  });
 
-      <EuiInMemoryTable<Category>
-        compressed
-        items={categories}
-        columns={columns}
-        selection={selectionValue}
-        itemId="key"
-        onTableChange={onTableChange}
-        pagination={pagination}
-        sorting={sorting}
-        data-test-subj="aiopsLogPatternsTable"
-        itemIdToExpandedRowMap={itemIdToExpandedRowMap}
-        rowProps={(category) => {
-          return enableRowActions
-            ? {
-                onClick: () => {
-                  if (category.key === pinnedCategory?.key) {
-                    setPinnedCategory(null);
-                  } else {
-                    setPinnedCategory(category);
-                  }
-                },
-                onMouseEnter: () => {
-                  setSelectedCategory(category);
-                },
-                onMouseLeave: () => {
-                  setSelectedCategory(null);
-                },
-                style: getRowStyle(category),
-              }
-            : undefined;
-        }}
-      />
-    </>
+  return (
+    <EuiInMemoryTable<Category>
+      compressed
+      items={categories}
+      columns={columns}
+      selection={selectionValue}
+      itemId="key"
+      onTableChange={onTableChange}
+      pagination={pagination}
+      sorting={sorting}
```

Comment:
The direction of sort on the Count column isn't being persisted when the view refreshes.

Reply:
Fixed in https://github.com/elastic/kibana/pull/178916/commits/c890187dbc565c8ac852acf472e58c57730c2fb9